api_tests/nodes/views/test_node_detail.py

```diff
@@ -61,6 +61,13 @@ def project_private(self, user):
             is_public=False,
             creator=user)
 
+    @pytest.fixture()
+    def wiki_private(self, user, project_private):
+        with mock.patch('osf.models.AbstractNode.update_search'):
+            wiki_page = WikiFactory(node=project_private, user=user)
```

Comment:
specify a name

Reply:
The test isn't failing and it doesn't seem relevant to this PR -- I can't remember why it seemed worth noting during group CR, can you?

Reply:
Probabilistic failure. Because when you don't specify a name there is a 1/x chance that "home" will be chosen as the name, which will conflict with the default wiki created for all nodes. Then the test will fail.